### ISSUE TITLE
How to run 2 stacks on same VPS with 2 different ips? On vultr or DigitalOcean

### ISSUE THREAD

**Issue by 9zxytlcs** [OlivierGaland/CashFactory#42]
I have setup some stacks on vultr Ubuntu VPS
One of my VPS host have 2 ips, the default along eith another fixed attached IP.

How to run 2 stacks of the project one with default vps IP the other with my second available IP? 

I'm using Vultr.com

**Comment by engageub** [comment]
You may have a look at this which supports both multiproxy with multi account.
https://github.com/engageub/InternetIncome
Thank you

**Comment by nowheel** [comment]
you get banned from various platform if run on vps

**Comment by engageub** [comment]
No. Most of the bandwidth sharing apps support datacenter IP address. So you have to chose accordingly which apps support datacenter IP address.

For more information, please see the link below. 
https://github.com/engageub/InternetIncome#judge-comparison-of-different-apps-based-on-proxy-type


Thank you

**Comment by engageub** [comment]
For multi IP, you can see the link here also if you are looking for this. https://github.com/engageub/InternetIncome/issues/9

**Comment by rennode** [comment]
> No. Most of the bandwidth sharing apps support datacenter IP address. So you have to choose accordingly which apps support datacenter IP address.
> 
> For more information, please see the link below. https://github.com/engageub/InternetIncome#judge-comparison-of-different-apps-based-on-proxy-type
> 
> Thank you
can you tell me which bandwidth sharing supports datacenter IP?

**Comment by engageub** [comment]
> > No. Most of the bandwidth sharing apps support datacenter IP address. So you have to choose accordingly which apps support datacenter IP address.
> > For more information, please see the link below. https://github.com/engageub/InternetIncome#judge-comparison-of-different-apps-based-on-proxy-type
> > Thank you
> > can you tell me which bandwidth sharing supports datacenter IP?

They are already present in the table mentioned in the link.
